static/css/place/change_place_search_bar.scss

```diff
@@ -0,0 +1,161 @@
+/**
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/** Styles for "Change Place" search bar on place pages */
+
+// "Change Place" search bar colors
+$GM3-sys-light-primary: #0b57d0;
+$GM3-sys-light-surface-container-low: #f8fafd;
+$GM3-sys-light-on-surface: #1f1f1f;
+
+#place-change-ui {
+  flex-grow: 1;
+  margin-bottom: 4px;
+}
+
+#place-search-container {
+  align-items: flex-end;
+  display: flex;
+  height: 100%;
+  width: 100%;
+}
+
+#change-place {
+  display: flex;
+  height: fit-content;
+  justify-content: right;
```

Comment:
Can you try left-aligning when in mobile? The right alignment doesn't look quite right to me

Reply:
Tried many CSS/flexbox variations to get left-aligning on wrap, ended up setting left-align behavior using a `@media` rule with a breakpoint found empirically. PTAL and let me know if you dislike this approach.